pages/common/lab.md

```diff
@@ -0,0 +1,19 @@
+# lab
+
+> A CLI for Gitlab.
+
+- Create a merge request and specify remote and target branch:
+
+`lab mr create {{remote}} {{target_branch}}`
+
+- Create a pull request on a github repo:
+
+`lab pull-request`
+
+- Open the repo in your browser:
+
+`lab browse`
+
+- After a fork, compare your branch with the origin branch in your browser:
```

Comment:
Is it compulsory to have a fork ? Can't it work when the repo is cloned ?

Reply:
Nop, I was thinking that my use case was the most usefull. I'll update